### ISSUE TITLE
Exits early if there are any errors in one particular command

### ISSUE THREAD

**Issue by klauern** [FiloSottile/gorebuild#12]
I am running this tool to rebuild, but it fails really early when trying to process 'Go-Package-Store':

```sh
08:46 $ gorebuild
github.com/dustin/go-humanize
github.com/google/go-github/github
github.com/shurcooL/Go-Package-Store/presenter/github
github.com/shurcooL/Go-Package-Store/cmd/Go-Package-Store
2017/03/06 08:46:06 rename /tmp/gorebuild465739942/Go-Package-Store /home/klauer/dev/go/bin/Go-Package-Store: invalid cross-device link
```

There may be another issue with just dealing with `Go-Package-Store`, but I would expect that even if it fails with that one for some reason, it should continue to rebuild the rest of the packages anyway.

**Comment by klauern** [comment]
NOw, I do have a workaround, but I've been seeing this issue with this one for so long that I thought it worth pointing out that my default way to run this is

```
gorebuild | xargs go install
```

or even

```
gorebuild | xargs -I {} go get -u {}
```

so I can pass over those errors.

**Comment by dmitshur** [comment]
> There may be another issue with just dealing with `Go-Package-Store`

Is there an issue with Go Package Store itself, or is it an issue with how you have it installed? If there's something to fix in Go Package Store, please [report an issue](https://github.com/shurcooL/Go-Package-Store/issues) if you don't mind so I can fix it. I'm not aware of issues, and I tested that latest master version compiles successfully.

Based on the error message `2017/03/06 08:46:06 rename /tmp/gorebuild465739942/Go-Package-Store /home/klauer/dev/go/bin/Go-Package-Store: invalid cross-device link`, it sounds like something related to `gorebuild` or your filesystem or GOPATH configuration (permissions, volumes, hard/soft links, etc.).

**Comment by klauern** [comment]
No, there's not an issue with that tool itself (I use it pretty regularly).  I can build `Go-Package-Store` using `go get`, but not this tool.
